From 5e5b46ec81534f34dffcca9795c528c4074a563f Mon Sep 17 00:00:00 2001
From: Alexey Dobriyan <adobriyan@openvz.org>
Date: Thu, 28 Feb 2008 19:24:52 +0300
Subject: [PATCH 03/48] SLUB: drop "inline" from __flush_cpu_slab() prototype

3.4 gcc sometimes treats as error the following sequence:

	static inline foo();

		foo();

	static inline foo()
	{
		...
	}
---
 mm/slub.c |    2 +-
 1 files changed, 1 insertions(+), 1 deletions(-)

diff --git a/mm/slub.c b/mm/slub.c
index 3317f93..b251d42 100644
--- a/mm/slub.c
+++ b/mm/slub.c
@@ -346,7 +346,7 @@ unsigned long ub_cache_growth(struct kmem_cache *cachep)
 	return atomic_read(&cachep->grown) << cachep->order;
 }
 
-static inline void __flush_cpu_slab(struct kmem_cache *s, int cpu);
+static void __flush_cpu_slab(struct kmem_cache *s, int cpu);
 
 static int kmem_cache_walk_page(struct page *pg, struct kmem_cache *s,
 		int (*fun)(void *))
-- 
1.5.4.3

